torch/csrc/jit/async.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "torch/csrc/jit/ivalue.h"
+#include "torch/csrc/jit/script/module.h"
+
+namespace torch { namespace jit { namespace async {
+
+struct Future {
+  explicit Future(IValue result) : result(result), ready(true) {}
+  IValue result;
+  bool ready = false;
+  IValue get() const {
+    JIT_ASSERT(ready);
+    return result;
+  }
+};
+
+Future fork(script::Module &sm, std::vector<IValue> inputs) {
```

Comment:
This isn't called, as far as I can tell, and I don't think its signature is correct. So we should probably just not check it in.